.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
+  "attr-bans": [
+    "align",
+    "background",
```

Comment:
![Screenshot 2022-08-16 at 15 25 55](https://user-images.githubusercontent.com/84275496/184879262-1e16ec24-1b7c-4a22-aa6f-0995e554c032.png)

Reply:
🥲

Reply:
Please, add only files with your code in future PRs

Reply:
Noted 🙂  Am now using git add before commit and without tags -am